SQLFrontend/ClientContext.h

```diff
@@ -0,0 +1,59 @@
+#ifndef CLIENTCONTEXT_H
+#define CLIENTCONTEXT_H
+
+#include "gen-cpp/mapd_types.h"
+
+using namespace ::apache::thrift;
+using namespace ::apache::thrift::protocol;
+using namespace ::apache::thrift::transport;
+
+static constexpr char const* INVALID_SESSION_ID("");
+static constexpr char const* MAPD_ROOT_USER("mapd");
+static constexpr char const* MAPD_DEFAULT_ROOT_USER_ROLE("mapd_default_suser_role");
```

Comment:
Why not std::string?

Reply:
They can be std::string if it works.  

When I did the first pass conversion,  I wasn't entirely sure whether any of the C-style functions were dependent on these definitions from elsewhere at first glance.  I figured anywhere that took a std::string would work automatically and the C functions would still work.  If the std::string works without issue or makes the code better, I can make the change.

Reply:
Looks like INVALID_SESSION_ID only compared against a string in ClientContext.  MAPD_DEFAULT_ROOT_USER_ROLE not really found in SQLFrontend/* and MAPD_ROOT_USER used once in some informational privileges message.  They're probably safe for conversion.  I'll change them now.